### ISSUE TITLE
sharedassets on latest version

### ISSUE THREAD

**Issue by Dragoonglue** [hatoving/MonumentValley#1]
The `PHONE_PATCHES_LATEST.zip` file lacks the necessary xdelta patches to extract any of the sharedassets files. Also, for whatever reason the `applyPatchAndroidLatest.bat` file tries to open the `PHONE_PATCHES` folder instead of `PHONE_PATCHES_LATEST`.

**Comment by hatoving** [comment]
Well, first off, do you have the right files in place? (You need the APK files in that folder and the obb files in "assets/bin/Data" I believe)
And second off, I'm so sorry about the batch file lmfao
I was very tired and I just wanted to get over it as soon as possible.

**Comment by Dragoonglue** [comment]
Yes, I have everything in the right place. I could generate the `levelX` files without issue, and they are derived from the same `data.unity3d` file that the `sharedassetsX.assets` files are supposed to come from. It's just that the `PHONE_PATCHES_LATEST.zip` in the releases section doesn't include the `sharedassetsX.assets.xdelta` patches that the batch file is trying to open, and because of that I can't generate the `sharedassetsX.assets` files.

**Comment by hatoving** [comment]
hey there, sorry for not answering. I've gotten sick and haven't had much time to correct these. there should be a new zip file at the Releases page, could you try that one out? Thanks

**Comment by Dragoonglue** [comment]
Hello, thanks for answering. Unfortunately, now I get checksum errors and can't even generate the `levelX` files, when I could before.

**Comment by hatoving** [comment]
that's weird, i just tested it now by myself and it works just fine.
i'll suggest a couple of things:
- clean out the patch files and start from scratch, maybe that'll help?
- absolutely make sure that you have the latest version (it should be v3.3.104)
- try the older version (v2.5.16) or the latest version of the pc release
if none of those work or you just don't want to pay for the other versions, i'll figure somethin out
